pkg/acceptance/bettertestspoc/config/model/warehouse_model_gen.go

```diff
@@ -75,6 +76,11 @@ func (w *WarehouseModel) WithEnableQueryAcceleration(enableQueryAcceleration str
 	return w
 }
 
+func (w *WarehouseModel) WithFullyQualifiedName(fullyQualifiedName string) *WarehouseModel {
```

Comment:
Just a question: do we have in the generator docs to omit fields that are computed only? (it does not make sense to have fully qualified in the model, so asking to make sure we have it documented to remove it in the future)